assets/src/components/ladder.tsx

```diff
@@ -2,13 +2,28 @@ import React, { useContext } from "react"
 import DispatchContext from "../contexts/dispatchContext"
 import { Timepoint, Vehicle, VehicleId } from "../skate"
 import { selectVehicle } from "../state"
+import VehicleIcon from "./vehicleIcon"
 
 // Timepoints come from the API in the ZeroToOne direction
 export enum LadderDirection {
   ZeroToOne,
   OneToZero,
 }
 
+export enum VehicleDirection {
+  Up,
+  Down,
+}
+
+export const vehicleDirectionOnLadder = (
+  vehicle: Vehicle,
+  ladderDirection: LadderDirection
+) =>
```

Comment:
I think a return type here would help.
```suggestion
): VehicleDirection =>
```